SPECS/kernel/config_aarch64

```diff
@@ -671,11 +761,16 @@ CONFIG_HAVE_MOD_ARCH_SPECIFIC=y
 CONFIG_MODULES_USE_ELF_RELA=y
 CONFIG_ARCH_HAS_ELF_RANDOMIZE=y
 CONFIG_HAVE_ARCH_MMAP_RND_BITS=y
-CONFIG_ARCH_MMAP_RND_BITS=32
+CONFIG_ARCH_MMAP_RND_BITS=18
```

Comment:
This config has been identified by the kernel config hardening checker and should be set to 32.

Reply:
updated.